### PR TITLE
W5-push notification some new changes

### PR DESCRIPTION
**Pull request by dev-siris**
@syuraj @shrestha2lt8  sir naya changes haru xa

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -108,6 +110,13 @@ function App() {
 
 	useEffect(() => {
 		SplashScreen.hide()
+
+		const channel = new firebase.notifications.Android.Channel(
```
Comment (syuraj) [inline app.js]: @chetannn is this a test code?

Comment (dev-siris) [reply app.js]: @syuraj  sir haina.. name matra test rakheko..

Comment (syuraj) [reply app.js]: ok, probably we need a better name e.g. `Top news channel`

Comment (dev-siris) [reply app.js]: @syuraj  sir hunxa

**Pull request merged**